README.md

```diff
@@ -185,7 +185,8 @@ Here, we describe the requests and responses as function arguments and return va
   * __Arguments__:
     * `Height (int64)`: The block height that ended
   * __Returns__:
-    * `Diffs ([]Validator)`: Changed validators with new voting powers (0 to remove)
+    * `Changes ([]Validator)`: Changed validators with new voting powers (0 to remove)
+    * `ConsensusParamChanges (ConsensusParams)`: Communicate changes to consensus-critical time/size parameters to tendermint core
```

Comment:
I'd remove `Communicate`. Just `Changes to ...`

Reply:
If we're going to rename this, lets get more specific.  Like ValidatorUpdates or something.

Reply:
s/Changes/Updates/g, more canonical.